### PR TITLE
fix/CSI 3571 node id length

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: node/pkg/driver/node_utils_test.go [hunk 1]
```diff
@@ -329,33 +329,33 @@ func TestGenerateNodeID(t *testing.T) {
...
-			expNodeId: fmt.Sprintf("%s;;%s;%s", hostName, strings.Join(fcWWNs, ":"), iscsiIQN),
+			expNodeId: fmt.Sprintf("%s;;%s;%s", hostName, strings.Join(fcWWNs[:2], ":"), iscsiIQN),
 		},
 		{name: "fail long hostName on nvme port",
 			hostName: longHostName,
 			nvmeNQN:  nvmeNQN,
 			fcWWNs:   []string{},
 			iscsiIQN: "",
-			expErr:   errors.New(fmt.Sprintf("could not fit any ports in node id: %s;, length limit: 192", longHostName)),
+			expErr:   errors.New(fmt.Sprintf("could not fit any ports in node id: %s;, length limit: 128", longHostName)),
```
Comment (oriyarde) [inline node/pkg/driver/node_utils_test.go:344]: while you're at it, could we use the tested const here, instead of maintaining it separately?

Comment (ArbelNathan) [reply node/pkg/driver/node_utils_test.go]: done

Comment (oriyarde) [reply node/pkg/driver/node_utils_test.go]: should we import it, or maintain it separately?
IMO the const value is not "logic", and the tests should check logic, so it's ok to import it

Comment (ArbelNathan) [reply node/pkg/driver/node_utils_test.go]: done

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**